controllers/BlogController.php

```diff
@@ -0,0 +1,59 @@
+<?php namespace SureSoftware\PowerBlog\Controllers;
+
+use Illuminate\Routing\Controller;
+use Rainlab\Blog\Models\Post;
+use SureSoftware\PowerBlog\Models\Settings;
+
+/**
+ * Blog API Controller
+ */
+class BlogController extends Controller
+{
+    /**
+     * Return latest 3 blog posts.
+     *
+     * @return \Illuminate\Http\JsonResponse
+     */
+    public function getPublished()
+    {
+        // Check whether API is enabled.
+        if (Settings::get('enable_api', false) != true) {
+            return response()->json('Currently unavailable.', 404);
+        }
+
+        // Return the latest 3 blog posts
+        $posts = Post::isPublished()
```

Comment:
@kodieupton please make this a parameter on the API, and default it to 10

Reply:
@Humni Change has been made to allow for changing limit on api request